### ISSUE TITLE
QAQC compile doi T/F issue

### ISSUE THREAD

**Issue by jb388** [International-Soil-Radiocarbon-Database/ISRaD#56]
@alkalifly @ShaneStoner @greymonroe @aahoyt 
Just moving this email conversation here. 

The error message is:
Error in if ((!(RCurl::url.exists(paste0("https://www.doi.org/", dois[d])) |  : 
  missing value where TRUE/FALSE needed

Interestingly, the QAQC works on stand-alone files without throwing this error.

**Comment by jb388** [comment]
Just a thought, but are we trimming the header rows twice when we call 'QAQC' from within 'compile'? I suspect this is creating "NA"s that are causing the above error.

**Comment by ShaneStoner** [comment]
> Just a thought, but are we trimming the header rows twice when we call 'QAQC' from within 'compile'? I suspect this is creating "NA"s that are causing the above error.

Are you referring to my last update to the QAQC function? I personally didn't add any clipping, just added 3 to the row number reported by the function so that it matches what you see when you open the excel file, i.e. the function will say "error on row 40" when the error is actually on the .xlsx row 43". Nothing too special.

I noticed the same thing about single files passing but failing during compile. This will have to be a job for someone other than me, I'm not sure how it's working differently between the two files at the moment. But since everything passes QAQC singularly, I think it is still okay that I blocked the QAQC during the compile in the dev branch.

**Comment by ShaneStoner** [comment]
Update: after looking at the code some more, I found that it runs QAQC on every file individually, then, at the end, attempts to run QAQC on the compiled ISRaD list file. That makes more sense that the error occurs when it does, right at the end after the main compilation processes have been completed. This is the instance of QAQC that I have commented out. I'm not sure what this was supposed to accomplish but I'd be curious to know! It isn't in the master branch version of that function.
